docs/CONTRIBUTING.md

```diff
@@ -324,13 +324,6 @@ value. The `core.IsAnyNilOrZero` function can check these cases.
 Senders must check that timestamps are non-zero before sending them. Receivers
 must check that timestamps are non-zero before accepting them.
 
-# Rounding time in DB
```

Comment:
I think this bit of documentation can/should be left in. We're still doing this, just not at each callsite.